src/fa-screenshot.sh

```diff
@@ -190,14 +171,10 @@ printOut "-- Checking image properties"
 printFileAttribs
 
 printOut "-- Resizing screenshot"
-convert=`which convert`
-$convert -geometry 1600x ${absFile} ${absFile}
-
-printOut "-- Compressing png"
-compressUsingTinyPNG
+magick=`which magick`
+$magick ${absFile} -geometry 1600x ${absFile}
```

Comment:
```suggestion
$magick convert ${absFile} -geometry 1600x ${absFile}
```
pretty sure we need the tool name in here

Reply:
I thought so too, but it complains if you do.